### DISCUSSION TITLE
Is it possible to transcribe & translate at the same time? Or to translate a completed transcription?

### DISCUSSION THREAD

**Discussion by mayeaux** [openai/whisper#420] [Q&A] [answered]
To me, it seems that the first generation of the transcript is the much more resource intensive process versus the translation process, therefore if I have non-English audio would it be possible to create the transcription and the English translation simultaneously, rather than run two separate resource intensive processes? Likewise, if I have the transcript would it be possible to use the model to translate that rather than run through the whole transcription process again? Thanks, love the module!

**Comment by jianfch** [comment, marked as answer]
Transcription tends to take longer than translation when `language` tag does not match actual language spoken in the audio because it uses temperature fallback (by default) more often when it is unsure of the results. 
As for transcribing and translating at the same time, see [comment](https://github.com/openai/whisper/discussions/386#discussioncomment-3936570).
